src/AssetManager/Module.php

```diff
@@ -83,6 +86,17 @@ public function onBootstrap(EventInterface $event)
         $eventManager->attach(MvcEvent::EVENT_DISPATCH_ERROR, $callback, $priority);
     }
 
+    public function getViewHelperConfig()
+    {
+        return array(
+            'factories' => array(
```

Comment:
Also move the factory to the config with the other factories. Sorry. I thought you knew :)